packages/service-clients/tinylicious-client/api-report/tinylicious-client.public.api.md

```diff
@@ -9,16 +9,8 @@ import { ContainerSchema } from '@fluidframework/fluid-static';
 import { IFluidContainer } from '@fluidframework/fluid-static';
 import { IMember } from '@fluidframework/fluid-static';
 import { IServiceAudience } from '@fluidframework/fluid-static';
-import { ITelemetryBaseEvent } from '@fluidframework/core-interfaces';
 import { ITelemetryBaseLogger } from '@fluidframework/core-interfaces';
 import { ITokenProvider } from '@fluidframework/routerlicious-driver';
 import { IUser } from '@fluidframework/driver-definitions';
 
-export { ITelemetryBaseEvent }
-
-export { ITelemetryBaseLogger }
-
-export { TinyliciousClient }
-export default TinyliciousClient;
```

Comment:
The exports of `TinyliciousClient` should not have appeared in any of these reports, since it is tagged as `@internal`.

I've filed a bug with API-Extractor about this issue: https://github.com/microsoft/rushstack/issues/4775